.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "_data/specifications"]
+	path = _data/specifications
+	url = https://github.com/BioSchemas/specifications.git
```

Comment:
@AlasdairGray do we still need this? If we pull straight from the specifications repo using requests, I guess we don't need a local copy with versioning in this repo?

Reply:
Yes the submodule definition can go as we can't process the json files without first manipulating them using the script I developed here.